lametro/management/commands/generate_tag_analytics.py

```diff
@@ -49,31 +66,48 @@ def generate_tag_analytics(self):
         csv_string = StringIO()
         writer = csv.writer(csv_string, delimiter=',')
         writer.writerow(
-            ('File ID',
-             'Identifier',
+            ('File GUID',
+             'File ID'
+             'Matter ID',
              'File Name',
              'Last Action Date',
              'Tag',
+             'Tag GUID',
              'Tag Classification')
         )
 
         self.stdout.write('\nGenerating tag analytics...')
 
         for bill in tqdm(LAMetroBill.objects.iterator(chunk_size=200)):
+            matter_id = bill.api_representation['MatterId']
```

Comment:
You can parse this out of the API source rather than hitting the API. :-) 

```
>>> b.api_source
<BillSource: https://webapi.legistar.com/v1/metro/matters/5627>
```